sickbeard/webserve.py

```diff
@@ -205,6 +205,10 @@ def render(self, *args, **kwargs):
             kwargs['title'] = '500'
             kwargs['header'] = 'Mako Error'
             kwargs['backtrace'] = RichTraceback()
+            for (filename, lineno, function, line) in kwargs['backtrace'].traceback:
+                print("File %s, line %s, in %s" % (filename, lineno, function))
+                print(line, "\n")
+            print("%s: %s" % (str(kwargs['backtrace'].error.__class__.__name__), kwargs['backtrace'].error))
```

Comment:
@labrys i'd like to keep these, but then as debug loggers, for mako debugging. What you think?